en/Plugins/Random note.md

```diff
@@ -1 +1,3 @@
-Random note does just that, opens a note at random from your Vault. Many people find this useful for surfacing thoughts they had not considered in a long time, or spot checking for things that they may have forgotten to link. 
\ No newline at end of file
+Random note does just that, opens a note at random from your Vault. Many people find this useful for surfacing thoughts they had not considered in a long time, or spot checking for things that they may have forgotten to link.
+
+After you enable the Random Note plugin, you can create a [[Custom hotkeys|custom hotkey]] that opens a note a random.
```

Comment:
"that opens a random" -> "that opens a random note"?

Reply:
Thanks, I added the missing 't' in the last commit.

Reply:
Ahh I see, I misread that.